Dockerfile

```diff
@@ -33,6 +33,10 @@ RUN rvm install jruby-${JRUBY_VERSION} && gem update --system \
     && gem install bundler -v 2.4.6 \
     && gem install rake -v 13.0.6
 
+RUN alternatives --install /usr/bin/java java /usr/lib/jvm/java-1.8.0-openjdk/bin/java 1
+RUN update-alternatives --set java /usr/lib/jvm/java-1.8.0-openjdk/bin/java
+RUN export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
+
```

Comment:
I encounter some errors when using these commands in the barebone images:
```
sh-5.1# alternatives --install /usr/bin/java java /usr/lib/jvm/java-1.8.0-openjdk/bin/java 1
sh-5.1# update-alternatives --set java /usr/lib/jvm/java-1.8.0-openjdk/bin/java
sh-5.1# export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
sh-5.1# java
sh: java: command not found
sh-5.1# keytool
sh: keytool: command not found
sh-5.1# 
```

Please consider if the LCM ruby still have same errors.